docs/connectors/sources/s3.md

```diff
@@ -0,0 +1,84 @@
+# AWS S3 input connector
+
+:::note
+This page describes configuration options specific to the S3 input connector.
+See [top-level connector documentation](/connectors/) for general information
+about configuring input and output connectors.
+:::
+
+The AWS S3 input connector is used to load data from an S3 bucket to a Feldera table.
+It can be configured to load a single object or multiple objects selected based on a
+common S3 prefix.
+
+:::note
+When accessing an S3 bucket that stores data in the Delta Lake format, consider
+using the [Delta Lake connector](/connectors/source/delta) instead.
+:::
+
+## Configuration options
+
+| Property                      | Type   | Default    | Description   |
+|-------------------------------|--------|------------|---------------|
+| `aws_access_key_id`           | string |            | AWS Access Key id. This property must be specified unless `no_sign_request` is set to `true`.|
+| `aws_secret_access_key`       | string |            | Secret Access Key. This property must be specified unless `no_sign_request` is set to `true`.|
+| `no_sign_request`             | bool   | `false`    | Do not sign requests. This is equivalent to the `--no-sign-request` flag in the AWS CLI. |
+| `key`                         | string |            | Read a single object specified by a key. |
```

Comment:
It's either specify a key or a prefix right? If so, would be good to clarify that here.